### ISSUE TITLE
E251 false positive

### ISSUE THREAD

**Issue by myint** [PyCQA/pycodestyle#70]
This is from the `copy` module.

``` python
d[type(None)] = _deepcopy_atomic
```

```
copy_example.py:1:14: E251 no spaces around keyword / parameter equals
copy_example.py:1:16: E251 no spaces around keyword / parameter equals
```
